lops/lop-domain-a72.dts

```diff
@@ -26,12 +26,12 @@
                 lop_2 {
                         // modify to "nothing", is a remove operation
                         compatible = "system-device-tree-v1,lop,modify";
-                        modify = "/cpus_microblaze1::";
+                        modify = "/cpus-cluster@1::";
```

Comment:
System device-tree now generating the microblaze cpu node names as cpus_microblaze1 and cpus_microblaze2, No need of this change

Reply:
how recent is this? my dtg++ output did not have this

Reply:
I have reported this issue last week and it got fixed Couple of days back 18th Nov it got fixed) Please do git pull on your system device-tree repo....